extra/piccolo_admin

```diff
@@ -2,6 +2,9 @@ server {
     listen 80;
     server_name piccolo_admin;
 
+    // By default Nginx only allows us to upload very small files.
+    client_max_body_size 20M;
```

Comment:
You should add a `#` at the beginning of the comment or remove it, otherwise [nginx](http://nginx.org/en/docs/beginners_guide.html#conf_structure) failed to start.

Reply:
Good catch! Will fix it now.